api/database/migrations/2023_02_01_194314_connect_teams_pools.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+use App\Models\Team;
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // this Team is seeded in TeamSeeder.php, it is essential this team exists BEFORE migrating so the updateOrCreate is run to make sure
+        Team::updateOrCreate(
+            [
+                'name' => 'digital-community-management',
+            ],
+            [
+                'display_name' => [
+                    'en' => 'Digital Community Management',
+                    'fr' => 'Gestion de la collectivité numérique',
+                ],
+            ],
+        );
+
+        Schema::table('pools', function (Blueprint $table) {
+            $table->uuid('team_id')->references('id')->on('teams')->nullable()->default(null);
```

Comment:
I think we should have some on delete logic here in case a team is deleted. Unless we want to handle this in the controller?

```php
$table->uuid('team_id')
  ->references('id')
  ->on('teams')
  ->nullable()
  ->default(null)
  ->cascadeOnDelete();
```

Reply:
I don't think we do this anywhere else, so I'd rather spin off a discussion as to whether we want to implement this across the database